src/test/java/org/folio/rest/impl/GOBIIntegrationServiceResourceImplTest.java

```diff
@@ -985,6 +984,7 @@ private void verifyRequiredFieldsAreMapped(CompositePurchaseOrder compPO){
     assertNotNull(poLine.getOrderFormat());
     assertNotNull(poLine.getSource());
     assertNotNull(poLine.getTitleOrPackage());
+    assertNotNull(compPO.getAcqUnitIds());
```

Comment:
please change this assertion to check if this array is not empty

Reply:
done.